src/components/cycle-section/cycle-section/ChallengeCard.js

```diff
@@ -0,0 +1,26 @@
+import Link from 'next/link';
+import { Card } from 'primereact/card';
+import { ScrollPanel } from 'primereact/scrollpanel';
+import styles from './CycleSection.module.scss';
+import { Accordion, AccordionTab } from 'primereact/accordion';
+
+const ChallengeCard = ({ cycle }) => {
+  return (
+    <Card
+      className={`${styles.cycleSectionCard} ${styles.challengeCard}`}
+      title="Desafío"
+    >
+      <ScrollPanel className={styles.scrollPanel}>
+        {cycle.challenge_description}
+      </ScrollPanel>
+      <Accordion className={styles.accordionHeader}>
+        <AccordionTab header="Repositorio base">
+          <Link href={cycle.boilerplate_url}>Repositorio en GitHub</Link>
+        </AccordionTab>
+        <AccordionTab header="Tu repositorio">N/A</AccordionTab>
```

Comment:
This N/A will show some message? If this will be a link could be as the above message link

Reply:
If it is not yet implemented, better delete it

Reply:
Done!